packages/more-info/src/components/MoreInfo.test.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+
+import MoreInfo from "./MoreInfo";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("More info", () => {
+  let addEventListenerMock, removeEventListenerMock, wrapper;
+
+  beforeEach(() => {
+    wrapper = shallow(<MoreInfo icon="icon">Test</MoreInfo>);
+    addEventListenerMock = jest.spyOn(document, "addEventListener");
+    removeEventListenerMock = jest.spyOn(document, "removeEventListener");
+  });
+
+  afterEach(() => {
+    addEventListenerMock.mockRestore();
+    removeEventListenerMock.mockRestore();
+  });
+
+  test("showTooltip() set tooltipVisible to true", () => {
+    expect(wrapper.state("tooltipVisible")).toBe(false);
+
+    wrapper.instance().showTooltip();
```

Comment:
I prefer to simulate the mouseover event instead of calling the internal method

Reply:
Ok! It was my first thought too. Going to change.